### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by alvaralmstedt** [YiYuDL/CRNNTL#1]
Installation does not work as described in README.md

```
(CRNNTL) swegene36:CRNNTL alvaralmstedt$ ll
total 16
drwxr-xr-x  11 alvaralmstedt  staff   352 Oct 26 15:30 CRNNTL
-rw-r--r--   1 alvaralmstedt  staff  1062 Oct 26 15:30 LICENSE
-rw-r--r--   1 alvaralmstedt  staff  2407 Oct 26 15:30 README.md
drwxr-xr-x  10 alvaralmstedt  staff   320 Oct 26 15:30 datasets
(CRNNTL) swegene36:CRNNTL alvaralmstedt$ pip install .
ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
```

**Comment by YiYuDL** [comment]
Thanks for your information! I have updated the new setup.py file.

**Comment by alvaralmstedt** [comment]
There is still some issue with the environment.yaml file

```
(base) edu221193:yi_yu alvaralmstedt$ cd CRNNTL/
(base) edu221193:CRNNTL alvaralmstedt$ ll
total 32
drwxr-xr-x  12 alvaralmstedt  staff   384 Nov  5 09:19 CRNNTL
-rw-r--r--   1 alvaralmstedt  staff  1062 Nov  5 09:19 LICENSE
-rw-r--r--   1 alvaralmstedt  staff  2392 Nov  5 09:19 README.md
drwxr-xr-x  10 alvaralmstedt  staff   320 Nov  5 09:19 datasets
-rw-r--r--   1 alvaralmstedt  staff   202 Nov  5 09:19 environment.yml
-rw-r--r--   1 alvaralmstedt  staff   206 Nov  5 09:19 setup.py
(base) edu221193:CRNNTL alvaralmstedt$ conda env create -f environment.yml
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - pytorch=1.2.0

```

Maybe some additional channel needs to be added?

**Comment by YiYuDL** [comment]
I have added the PyTorch channel in the environment. Thanks for your suggestion！

**Comment by alvaralmstedt** [comment]
Great but there are still versioning conflicts. Does it work to create the env on your computer?

```
Looking for: ['python=3.6', 'tensorflow=1.14.0', 'numpy=1.16.4', 'pandas=1.2.4', 'scikit-learn', 'pytorch=1.2.0', 'skorch=0.10.0', 'rdkit']


Encountered problems while solving:
  - package pandas-1.2.4-py37hb23ed4d_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed
```

Changing to python v3.7 does not fix it:

```
Encountered problems while solving:
  - package pandas-1.2.4-py37hb2f4e1b_0 requires numpy >=1.16.6,<2.0a0, but none of the providers can be installed
```

**Comment by YiYuDL** [comment]
Sorry for the mistake. I have changed the pandas into version 1.0.3 in the yml file. It worked on my computer. If more information needed, please tell me.

**Comment by alvaralmstedt** [comment]
Nice! The environment building now works. Now I have the next problem:
```
(CRNNTL) MacBook-Pro:CRNNTL alvaralmstedt$ python test.py
Traceback (most recent call last):
  File "test.py", line 7, in <module>
    from CRNNTL.preprocessing import data_pre
  File "/Users/alvaralmstedt/repos/course_students_HT2021/yi_yu/CRNNTL/CRNNTL/preprocessing.py", line 13, in <module>
    from cddd.inference import InferenceModel
ModuleNotFoundError: No module named 'cddd'
```
Looks like cddd is not included.. Do I need to install that separately somehow? I see you have the link to this repo but how exactly does this need to be installed in relation to CRNNTL? I tried cloning it inside your repo and then moving the cddd directory inside out so it could be reacched. The import then worked but broke at a later stage, probably due to me messing up the directory structure:

```
(CRNNTL) MacBook-Pro:CRNNTL alvaralmstedt$ python test.py
Consider installing the package zmq to utilize the InferenceServer class
number of molecules after cleaning 4177
number of molecules smiles changed Counter({True: 4176, False: 1})
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

W1112 10:17:27.123901 4491388416 lazy_loader.py:50]
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

Traceback (most recent call last):
  File "test.py", line 23, in <module>
    app.run(main)
  File "/Users/alvaralmstedt/miniconda3/envs/CRNNTL/lib/python3.6/site-packages/absl/app.py", line 312, in run
    _run_main(main, args)
  File "/Users/alvaralmstedt/miniconda3/envs/CRNNTL/lib/python3.6/site-packages/absl/app.py", line 258, in _run_main
    sys.exit(main(argv))
  File "test.py", line 15, in main
    X_fgfr1, DF_fgfr1 = data_pre(fgfr1_df)
  File "/Users/alvaralmstedt/repos/course_students_HT2021/yi_yu/CRNNTL/CRNNTL/preprocessing.py", line 45, in data_pre
    inference_model = InferenceModel()
  File "/Users/alvaralmstedt/repos/course_students_HT2021/yi_yu/CRNNTL/cddd/inference.py", line 102, in __init__
    self.hparams = create_hparams(flags)
  File "/Users/alvaralmstedt/repos/course_students_HT2021/yi_yu/CRNNTL/cddd/hyperparameters.py", line 119, in create_hparams
    hparams = hparams.parse_json(json.load(open(hparams_file_name)))
FileNotFoundError: [Errno 2] No such file or directory: '/Users/alvaralmstedt/repos/course_students_HT2021/yi_yu/CRNNTL/cddd/data/default_model/hparams.json'
```

This could be explained a bit better :)
